Misc/NEWS.d/next/Core and Builtins/2021-08-31-17-44-51.bpo-45020.ZPI_3L.rst

```diff
@@ -1,5 +1,4 @@
 Freeze all the modules that are imported during Python startup.  This gives
 us meaningful performance improvements to startup.  Also add a new command
 line option, "-X frozen_modules=[on|off]" to opt out of (or into) using
-frozen modules.  This defaults to "on" (or "off" if running in a local
-development environment, saving contributors some frustration).
+frozen modules.  This defaults to "on" (or "off" if it's a debug build).
```

Comment:
I still think that the best rule would be: default on when --with-optimizations is give; default off otherwise. Tests for the freezing mechanism must turn it on explicitly anyway.

Reply:
That's the plan.